tests/yam/migration/migration_unattended.pm

```diff
@@ -45,18 +45,22 @@ sub run {
         }
     }
 
-    # upload logs to know system state before migration
-    upload_logs("/boot/grub2/grub.cfg", failok => 1);
-    upload_folders(folders => '/etc/zypp/repos.d/');
-
     if (is_s390x) {
         assert_script_run("echo 'PermitRootLogin yes' > /etc/ssh/sshd_config.d/root.conf");
         enter_cmd '/usr/sbin/run_migration';
         reset_consoles;
         reconnect_mgmt_console;
     } else {
+        # disable timeout for migration grub menu
+        assert_script_run("sed -i 's/set timeout=[0-9]*/set timeout=-1/' /etc/grub.d/99_migration");
+        assert_script_run("grub2-mkconfig -o /boot/grub2/grub.cfg");
+        # upload logs to know system state before migration
+        upload_logs("/boot/grub2/grub.cfg", failok => 1);
+        upload_folders(folders => '/etc/zypp/repos.d/');
```

Comment:
and this one is not needed right? kind of accidental copy/paste I think.

Reply:
yes, I move the code for uploading (grub+repos) where it was originally. thanks.